ui.go

```diff
@@ -123,6 +128,10 @@ func (u *BasicUi) Output(message string) {
 	fmt.Fprint(u.Writer, "\n")
 }
 
+func (u *BasicUi) Print(message string) {
+	fmt.Fprint(u.Writer, message)
+}
```

Comment:
I'd rather keep `message` as is, as it keeps style with the other interface functions, but agree on other points.